src/Integration.Vsix/packages.lock.json

```diff
@@ -1533,6 +1533,9 @@
           "Microsoft.VisualStudio.Interop": "17.0.31902.203"
         }
       },
+      "copydependencies": {
```

Comment:
Why is it lowercase?

Reply:
no idea it's auto generated.

Reply:
Some other stuff is also lowercase (e.g. "sonarqube.client"). Since it doesn't affect anything other than cosmetics on an auto-generated file I don't consider it as a problem.